modules/cnv-importing-vmware-vm.adoc

```diff
@@ -24,6 +24,7 @@ If you try to import a virtual machine whose disk size is larger than the availa
 
 . In the {CNVProductName} web console, click *Workloads* -> *Virtual Machines*.
 . Click *Create Virtual Machine* and select *Import with Wizard*.
+
```

Comment:
This isn't necessary between steps (when there's no text or code block following the step) but it's not hurting anything,